pootle/apps/pootle_store/fields.py

```diff
@@ -152,65 +135,23 @@ def realpath(self):
         else:
             return self._store_tuple.realpath
```

Comment:
`_store_tuple` can be dropped too, now that it is not populated. `realpath` would need merging with `_get_realpath`.